### ISSUE TITLE
Accessibility: non-links link should be buttons

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#86]
All the non-link links that act like UI controls, for example the menu items "toggles", should be marked up as buttons. See core trac ticket https://core.trac.wordpress.org/ticket/26504 and related ones. For example:
```
<a class="item-edit" id="edit-1052" title="Edit Menu Item" href="#">Edit Menu Item</a>
should be
<button type="button" class="item-edit" id="edit-1052" title="Edit Menu Item">Edit Menu Item</button>
```
It's also easier for the JavaScript part, no need to attach a keyboard event, no need to preventDefault() and they can be navitely activated with Enter or Spacebar.
Additionally, when they control a panel/slider, they should have an aria-expanded attribute to be dynamically updated depending on the collapsed/expanded status

**Comment by celloexpressions** [comment]
Please add comments here if you find any additional instances of this. Previously: #75.

**Comment by ocean90** [comment]
The example was fixed in https://github.com/voldemortensen/menu-customizer/issues/93.

**Comment by afercia** [comment]
Noticed one more case: the `item-delete` links should be buttons. This would also allow to simplify the JavaScript part, for example:
```
var isDeleteBtn = $( e.target ).is( '.item-delete, .item-delete *' ),
	isAddNewBtn = $( e.target ).is( '.add-new-menu-item, .add-new-menu-item *' );
```
no more need for the asterisk... and would allow to drop `is()` and use `hasClass()` which is faster. Some eyes on the events bound would be recommended too.
